compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/Patterns.cpp

```diff
@@ -183,6 +183,60 @@ struct ConvertTensorFromElementsPattern
   }
 };
 
+/// Returns a sizes array with the dynamic dims.
+static SmallVector<Value> getDynamicTensorSizes(OpBuilder &builder,
+                                                Location loc, ShapedType stp,
```

Comment:
What does `stp` mean? Can we rename it to `type`?

Reply:
It meant Shaped Type, but changed it now